climpyrical/data.py

```diff
@@ -81,29 +83,63 @@ def read_data(data_path: str, required_keys: list = None) -> xr.Dataset:
         )
 
     if required_keys is None:
-        required_keys = ["rlat", "rlon"]
+        required_keys = ["rlat", "rlon", "lat", "lon"]
```

Comment:
Couldn't we just set these as the default rather than `None` in the first place?

Reply:
:+1: